### PR TITLE
update velero compatibility

### PR DESCRIPTION
**Pull request by cbodonnell**
Updates Velero version compatibility table to reflect support for the latest version 1.12.1.

Also updates, the troubleshooting topic to indicate that the issue in 1.10.3 and 1.11.0 has been resolved more recent releases.

### CODE REVIEW

**Review by sgalsaleh — DISMISSED** [review]

File: docs/enterprise/snapshots-troubleshooting-backup-restore.md [hunk 1]
```diff
@@ -203,4 +203,4 @@ The resource restore priority was changed in Velero 1.10.3 and 1.11.0, which lea
 
 #### Solution
 
-These warnings do not necessarily mean that the restore itself failed. The endpoints likely do exist as they are created by Kubernetes when the related Service resources were restored. However, to prevent encountering these warnings, use Velero version 1.10.2 or earlier.
+These warnings do not necessarily mean that the restore itself failed. The endpoints likely do exist as they are created by Kubernetes when the related Service resources were restored. This issue is resolved in Velero versions 1.11.1 and 1.12.0.
```
Comment (ajp-io) [inline docs/enterprise/snapshots-troubleshooting-backup-restore.md:206]: Should we just say to use 1.11.1 and later?

File: docs/enterprise/snapshots-understanding.md [hunk 2]
```diff
@@ -77,5 +77,5 @@ The following table lists which versions of Velero are compatible with each vers
 | KOTS version | Velero version |
 |------|-------------|
 | 1.15 to 1.20.2 | 1.2.0 |
-| 1.20.3 and later | 1.5.1 through 1.9.x |
-| 1.94.1 and later | 1.6.x through 1.10.2 |
+| 1.20.3 to 1.94.0 | 1.5.1 through 1.9.x |
+| 1.94.1 and later | 1.6.x through 1.12.x |
```
Comment (ajp-io) [inline docs/enterprise/snapshots-understanding.md:81]: We used to just say "1.6.x and later." I know there are no later versions right now, but do we want to say "and later" or "through 1.12.x"? This requires more updating, but it prevents us from saying a version works that we haven't tried yet.

Comment (cbodonnell) [reply docs/enterprise/snapshots-understanding.md:81]: We anticipate 1.13 to be the version where Restic support is deprecated, so we may not support that right away without additional work from our side.

Related SC: https://app.shortcut.com/replicated/story/85863/velero-support-for-kopia-and-or-csi-snapshots#activity-89333

**Review by ajp-io — APPROVED** [review]

**Comment by ajp-io** [comment]
Ready for @replicatedhq/replicated-docs

**Review by paigecalvert — APPROVED** [review]

**Pull request merged**